utils/changelog

```diff
@@ -28,7 +28,9 @@ import subprocess
 usage = "Usage: changelog [options] [<new version>]"
 parser = argparse.ArgumentParser(usage=usage)
 parser.add_argument("--tag", dest="tag",
-                    help="git tag")
+                    help="Start changelog from this git tag")
+parser.add_argument("--upto", dest="upto_tag", default="",
```

Comment:
This change adds `--upto`, but no `--last`.

Reply:
It should be '--upto'. Fixed.